src/core/vertex.js

```diff
@@ -269,15 +269,17 @@ p5.prototype.beginShape = function(kind) {
 
```

Comment:
This might have to be dealt with in the merge from "webgl-gsoc-2018" to "master" but worth noting now that this file is in the [shape folder](https://github.com/processing/p5.js/blob/master/src/core/shape/vertex.js) now in master

Reply:
I think this will show up as a conflict in github but will merge cleanly—that's what happened with other changes to files that had been moved.